### ISSUE TITLE
handle special characters in certs like é, ó

### ISSUE THREAD

**Issue by jedgalbraith** [nanopack/portal#36]
When a cert field has special characters like `Velavi Cosméticos` or `Niterói` portal responds with `500:{"error":"asn1: syntax error: PrintableString contains invalid character"}` when attempting to register a new cert.

**Comment by glinton** [comment]
So far, I have been unable to reproduce this. I have found where that error comes from though. A valid x509 key/cert are required. I generated a keypair using the following as subject entries and was still unable to get the error:

```
C=US
ST=étalia
L=mético
O=Velavi Cosméticos
OU=ét
CN=thing.com
```

I've updated the error returned to a 400 for that error though, as it seems to be a misconfigured/generated cert.

**Comment by jedgalbraith** [comment]
Oh, ok. Cool. That's probably what it was then. Now that it's a 400, the user will be able to deal with it. Thanks.

**Comment by danhunsaker** [comment]
It looks like the field was incorrectly encoded as ASN.1 PrintableString, when it should have been either ASN.1 UTF8String, or possibly ANS.1 T61String.

So yeah, incorrectly encoded cert.

**Comment by glinton** [comment]
Resolved with #39
